src/main/views/layout.njk

```diff
@@ -25,3 +25,5 @@
   {% include "includes/scripts.njk" %}
   {% block additional_scripts %}{% endblock %}
 {% endblock %}
+
```

Comment:
why add these lines?

Reply:
Was trying a solution within this file originally so will remove line left behind